src/main/java/com/google/sps/servlets/CivicServlet.java

```diff
@@ -0,0 +1,59 @@
+package com.google.sps.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.util.Map;
+import java.util.HashMap;
+import java.lang.reflect.Type;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.google.sps.data.UrlRequest;
+
+/** Servlet that gets civic information. */
+@WebServlet("/civic")
+public class CivicServlet extends HttpServlet {
+
+  private static final String API_KEY = "API_KEY";  // Insert the API_KEY here for testing.
+  private static final String API_PATH = "https://www.googleapis.com/civicinfo/v2/representatives";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String latitude = request.getParameter("lat");
+    String longitude = request.getParameter("lng");
+    Map<String, String> locationQueryParams = new HashMap<>();
+    locationQueryParams.put("lat", latitude);
+    locationQueryParams.put("lng", longitude);
+    String locationUrl = String.format("%s://%s:%s/location",
+        request.getScheme(), 
+        request.getServerName(), 
+        request.getServerPort());
+    String address = "";
+    try {
+      // Location URL doesn't work on devserver, so instead use hardcoded string.
```

Comment:
[Nit] I think this comment makes more sense if you put it in the catch block below.